nameko/testing/pytest.py

```diff
@@ -287,7 +287,9 @@ def socket_creator():
             '127.0.0.1', web_config_port)
         gr = eventlet.spawn(ws_app.run_forever)
         active_sockets.append((gr, ws_app))
-        return wait_for_sock()
+        socket = wait_for_sock()
+        socket.app = ws_app
```

Comment:
If we're going to attach the `ws_app` to the `socket` object, perhaps `make_virtual_socket` should pass it in at [construction time](https://github.com/onefinestay/nameko/blob/master/nameko/testing/websocket.py#L45)?

Also, I found it confusing that this variable is called `socket`, assuming it to be the underlying socket object. It is really a websocket proxy (although "proxy" is also a heavily used word with other meanings).